pallets/runtime/tests/src/storage.rs

```diff
@@ -950,3 +952,73 @@ macro_rules! assert_event_doesnt_exist {
         }));
     };
 }
+
+pub fn exec<C: Into<<TestStorage as frame_system::Config>::Call>>(
+    origin: <TestStorage as frame_system::Config>::Origin,
```

Comment:
Could you type alias `::Origin`, `::Call`, and `::AccountId` and use it throughout the module?